core/shared/protobuf_configmap_reconciler.go

```diff
@@ -72,10 +72,13 @@ func (p *protoBufConfigMapReconciler) createRequiredResources(runtimeInstance ap
 	if err != nil {
 		return nil, err
 	}
+	if protoBufConfigMap!=nil{
+	
 	if err := framework.SetOwner(runtimeInstance, p.Scheme, protoBufConfigMap); err != nil {
```

Comment:
Please format this file.

Reply:
This is in reference to [KOGITO-3779](https://issues.redhat.com/browse/KOGITO-3779)
Don't create protobuf-files config map when no protobuf files are available

Reply:
Thanks! Please, next time, add the JIRA to the PR description, so we can see what it's about. Thanks again for the PR.

Reply:
sure ,orry about that.Will definitely take care of it,in the future.

Reply:
Oh,so sorry about that,I will definitely take care of it in the future.